src/kata5.reachDestination.js

```diff
@@ -1,3 +1,6 @@
-const reachDestination = (distance, speed) => {};
+const reachDestination = (distance, speed) => {
+    result = Math.round(distance/speed * 2) / 2;
```

Comment:
Too much indentation on line 2 and 3, only one tab or 2 spaces is enough for better formatting.

And the same issue here as it was in one of the function you had in javascript basics.
Assigning to a variable which is not declared properly: ⬇️ here the result variable is not declared properly
````
result = Math.round((distance / speed) * 2) / 2;
``````
to declare a variable you have to use the const or the let keyword. 
(const when the variable is constant and won't be reassigned, let ONLY when you are sure the variable is going to be reassigned)